source/SIL.AppBuilder.Portal/common/prisma/schema.prisma

```diff
@@ -106,6 +106,8 @@ model OrganizationMembershipInvites {
   Redeemed       Boolean       @default(false)
   InvitedById    Int
   OrganizationId Int
+  Roles          String?
```

Comment:
Is there any particular reason why you chose to use a string? As best as I can tell based on where these are used elsewhere, you just have a CSV list of numbers. Since we are using PostgreSQL we can have an array as a field type. See `WorkflowDefinitions.WorkflowOptions` for an example.